packages/core-api/src/apis/definitions/IdentityApi.ts

```diff
@@ -30,6 +30,7 @@ export type IdentityApi = {
    */
   getUserId(): string;
 
+  // TODO: getProfile(): Promise<Profile> - We want this to be async when added, but needs more work.
```

Comment:
This TODO seemed to be dangling, and so moved it closer to the method it referred to, though I'm not sure if it's still valid any longer! (esp given it was below a different method, it may just have been lost?)

Reply:
I actually think that this TODO is to create said function which doesn't exist yet. Gonna leave it there for now I think! 👍